test/tests/test_ingress.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -euo pipefail
+
+ROOT=$(dirname $0)/../..
```

Comment:
while this test is good and tests that the ingress objects get updated when the relative Url and host parameters of the corresponding httpTrigger object gets updated.
but, i do feel like it'll be good to have another test that verifies that the ingress does works and the request gets routed to the correct relativeUrl,

Reply:
Yeah, I would have liked it too, but that means deploying an ingress controller. Will take in another change.

Reply:
ok, 👍